test-cases/artifacts/amazon2.yaml

```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
+ami_id_db_scylla: 'ami-099a8245f5daa82bf'
+append_scylla_setup_args: '--no-cpuscaling-setup'
```

Comment:
@enaydanov what's the error in cpuscaling setup? do we have scylla issue for that?

Reply:
There is no `cpupower.service` file available on Amazon Linux 2. Filed https://github.com/scylladb/scylla/issues/5977